myhpom/templates/myhpom/accounts/choose_network.html

```diff
@@ -139,11 +139,12 @@ <h4>
         $customProvider.on('keypress', function (e) {
             $fullSelect.val('null');
             $radios.prop('checked', false);
+            $('#id_healthcare_network').val('');
```

Comment:
Functionally it doesn't make much difference, but it'd be Nice to move these up in the scope so that you just have one `var $healthcareNetworkInput = $('#id_healthcare_network');` statement (same goes for the next change below) rather than hitting the DOM afresh each time you press a key.

Reply:
That makes sense. I'll do that and merge it.